contributors/design-proposals/scheduling/scheduling-policy.md

```diff
@@ -0,0 +1,781 @@
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "amd64"
+      beta.kubernetes.io/os: "Linux"
+    priorityClasseName: "bronze"
+```
+
+### Option2: first-seen wins, ever
+In this option the strategy is at SchedulingPolicy level.
+
+for instance, if we have the following two schedpols that apply:
+
+```yaml
+Kind: SchedulingPolicy
+metadata:
+  name: schedpol-a # first in alphabetic order
+spec:
+  required:
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64"]
+  allowed:
+    nodeSelectors:
+      disk: ["ssd"]
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "amd64"
+---
+Kind: SchedulingPolicy
+metadata:
+  name: schedpol-b # second in alphabetic order
+spec:
+  required:
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64", "i386"]
+      beta.kubernetes.io/os: ["Linux", "Windows"]
+    priorityClasseNames: ["bronze", "gold", "silver"]
+  allowed:
+    nodeSelectors:
+      disk: ["sata"]
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "i386"
+      beta.kubernetes.io/os: "Linux"
+    priorityClasseName: "bronze"
+```
+
+The merged applied schedpol will be:
+
+```yaml
+Kind: SchedulingPolicy
+metadata:
+  name: schedpol-a # first in alphabetic order
+spec:
+  required:
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64"]
+  allowed:
+    nodeSelectors:
+      disk: ["ssd"]
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "amd64"
+```
+
+### Option3: simple merge
+
+In this strategy, merge is performed on a first-seen-wins on second-level entries.
+for instance, if we have the following two schedpols that apply:
+
+```yaml
+Kind: SchedulingPolicy
+metadata:
+  name: schedpol-a # first in alphabetic order
+spec:
+  required:
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64"]
+  allowed:
+    nodeSelectors:
+      disk: ["ssd"]
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "amd64"
+---
+Kind: SchedulingPolicy
+metadata:
+  name: schedpol-b # second in alphabetic order
+spec:
+  required:
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64", "i386"]
+      beta.kubernetes.io/os: ["Linux", "Windows"]
+    priorityClasseNames: ["bronze", "gold", "silver"]
+  allowed:
+    nodeSelectors:
+      disk: ["sata"]
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "i386"
+      beta.kubernetes.io/os: "Linux"
+    priorityClasseName: "bronze"
+```
+
+The merged applied schedpol will be:
+
+```yaml
+Kind: SchedulingPolicy
+spec:
+  required:
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64"]
+    priorityClasseNames: ["bronze", "gold", "silver"]
+  allowed:
+    nodeSelectors:
+      disk: ["ssd"]
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "amd64"
+    priorityClasseName: "bronze"
+```
+## Default SchedulingPolicies
+
+### Restricted policy
+Here is a reasonable policy that might be allowed for any cluster without specific needs:
```

Comment:
I've already lost track of which fields are closed by default, and which are open. I'm worried this is too difficult to reason about.